server/routes/findInterventions/interventionDetailsPresenter.test.ts

```diff
@@ -173,22 +173,22 @@ three lines.`,
       describe('for a single-service intervention', () => {
         it('is the service category name', () => {
           expect(
-            linesForKey('Service type', {
+            linesForKey('Service category', {
               serviceCategories: [serviceCategoryFactory.build({ name: 'accommodation' })],
             })
           ).toEqual(['Accommodation'])
         })
       })
 
       describe('for a cohort intervention', () => {
-        it('has a pluralised key, and is a list of service category names', () => {
+        it('has a pluralised key, and is a list of service categories names', () => {
```

Comment:
this can stay as "list of service category names"

Reply:
changed